lib/grinder_sdk.dart

```diff
@@ -44,7 +44,7 @@ class Dart {
   /// Returns the stdout.
   static String run(String script,
       {List<String> arguments : const [], bool quiet: false,
-       String packageRoot, String workingDirectory, int vmNewGenHeapMB,
```

Comment:
One reason to keep it as deprecated would be to allow releasing this change without a breaking change version bump, but we have some other important breaking changes that need to happen soon anyways, so I'm not sure if we'll need to release before those or not anyway.  @devoncarew?